aiidalab_widgets_base/elns.py

```diff
@@ -15,30 +15,48 @@
 
 
 def connect_to_eln(eln_instance=None, **kwargs):
+
+    # Assumming that the connection can only be established to the ELNs
+    # with the stored configuration.
     try:
         with open(ELN_CONFIG, "r") as file:
             config = json.load(file)
     except (FileNotFoundError, json.JSONDecodeError, KeyError):
```

Comment:
Should the `write_to_config` have similar handling?

Reply:
but the only way this would fail is if it, for some reason is not able to create and/or write this new file

Reply:
ok, good point. Then looks fine to me. I do not know what will happen if this directory does not exists (i think there is now no way I can imagine this to happen ...)